adb-server/adbserver-desktop/src/main/java/com/kaspersky/adbserver/desktop/CommandExecutorImpl.kt

```diff
@@ -19,9 +20,9 @@ internal class CommandExecutorImpl(
         return when (command) {
             is CmdCommand -> cmdCommandPerformer.perform(command.body)
             is AdbCommand -> {
-                val adbCommand = "adb ${ adbServerPort?.let { "-P $adbServerPort " } ?: "" }-s $deviceName ${command.body}"
-                logger.d("The created adbCommand=$adbCommand")
-                cmdCommandPerformer.perform(adbCommand)
+                val adbCommand = "${ adbServerPort?.let { "-P $adbServerPort " } ?: "" }-s $deviceName ${command.body}"
+                logger.d("The created adbCommand=adb $adbCommand")
```

Comment:
Would be cool to print out actual adb path since it's configurable. Otherwise it may mislead someone during adb related issues debugging

Reply:
this good idea. I'm add print adb path to log at server startup.